Assets/Prefabs/Ped.prefab

```diff
@@ -444,12 +420,11 @@ MonoBehaviour:
   m_Script: {fileID: 11500000, guid: 62374ff6df2c74f9699075492ef7fdb9, type: 3}
   m_Name: 
   m_EditorClassIdentifier: 
-  m_health: 100
   m_onDamage:
     m_PersistentCalls:
       m_Calls:
-      - m_Target: {fileID: 114220037769542022}
-        m_MethodName: HandleDamageByDefault
+      - m_Target: {fileID: 114099438993920694}
+        m_MethodName: OnDamage
```

Comment:
why do you change damage logic for ped ?

Reply:
to prevent object to be destroyed and recreated every time, not it being disabled for a 30 seconds

Reply:
You didn't understand me. Don't change damage logic for **Ped**. He is not being destroyed and recreated.